### PR TITLE
Moar quotes

### PR DESCRIPTION
**Pull request by TheCurle**

### CODE REVIEW

**Review by ravenbuilder934 — COMMENTED** [review]
in your last PR you forgot CJDaBomb's nickname

**Comment by TheCurle** [comment]
he changes it often

**Review by ravenbuilder934 — CHANGES_REQUESTED** [review]
curle that's not how nicknames work, look at your own name

**Review by ravenbuilder934 — APPROVED** [review]

File: quotes/data.json [hunk 1]
```diff
@@ -190,12 +190,23 @@
...
             "roles": []
         },
         "Citrine": {
             "discriminator": "1282",
             "avatar": "https://cdn.discordapp.com/avatars/227118001206329344/ccd6eddd6a0e97fbfea9a6c7f2177fbf.png",
             "roles": []
+        },
+        "Max Hyper": {
+            "discriminator": "0310",
+            "avatar": "https://cdn.discordapp.com/avatars/217760439465934848/f42e3f3987f77bc74d0ff7eb05ed0b81.png",
```
Comment (ravenbuilder934) [inline quotes/data.json:203]: hmm actually this should prolly have `?size=128` on the end of it (maybe for lord samosa too)

**Review by sciwhiz12 — APPROVED** [review]
:shipit:

**Pull request merged**